### ISSUE TITLE
Request: Highlighting for Render-Markdown

### ISSUE THREAD

**Issue by nyaahilism** [nyoom-engineering/oxocarbon.nvim#95]
Hi!  I love this theme so much but I was wondering if there was any chance at adding in the highlights for render-markdown.nvim? If not it's all good!

Thank you for your time.

**Comment by swapxs** [comment]
yes this feature would be quite useful as most people take notes with some form of markdown. Just the basic would be a good start and in future we could improve if needed.

**Comment by nyaahilism** [comment]
Thank you so much!
